code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -167,6 +178,10 @@
 
 /mob/living/simple_animal/hostile/headcrab/poison/AttackingTarget()
 	. = ..()
-	var/mob/living/carbon/C = target
-	if(C.can_inject(null, 0, "head", 0))
-		C.reagents.add_reagent("lsd", 5)
+	if(iscarbon(target) && target.reagents)
+		var/inject_target = pick("chest", "head")
+		var/mob/living/carbon/C = target
+		if(C.stunned || C.can_inject(null, 0, inject_target, 0))
+			if(C.eye_blurry < 60)
+				C.AdjustEyeBlurry(10)
```

Comment:
Assuming someone gets 60 eyeblurry from this, how long does that take to wear off?

Reply:
Believe this is 20 seconds. Same as green spider

Reply:
60 is the same as green spider, yup